protocol/x/clob/types/keys.go

```diff
@@ -18,94 +18,91 @@ const (
 
 	// LongTermOrderPlacementKeyPrefix is the key to retrieve a long term order and information about
 	// when it was placed.
-	LongTermOrderPlacementKeyPrefix = PlacedStatefulOrderKeyPrefix + "long_term/"
+	LongTermOrderPlacementKeyPrefix = PlacedStatefulOrderKeyPrefix + "LT/"
 
 	// UntriggeredConditionalOrderKeyPrefix is the key to retrieve an untriggered conditional order and
 	// information about when it was placed.
-	UntriggeredConditionalOrderKeyPrefix = StatefulOrderKeyPrefix + "untriggered/conditional/"
+	UntriggeredConditionalOrderKeyPrefix = StatefulOrderKeyPrefix + "Untrig/"
 
 	// StatefulOrdersTimeSlicePrefix is the key to retrieve a unique list of the stateful orders that
 	// expire at a given timestamp, sorted by order ID.
-	StatefulOrdersTimeSlicePrefix = "stateful_orders_time_slice/"
+	StatefulOrdersTimeSlicePrefix = "ExpTime/"
 )
 
 // Transient Store
 const (
 	// NextStatefulOrderBlockTransactionIndexKey is the transient store key that stores the next
 	// transaction index to use for the next newly-placed stateful order.
-	NextStatefulOrderBlockTransactionIndexKey = "next_stateful_order_block_transaction_index"
+	NextStatefulOrderBlockTransactionIndexKey = "NextTxIdx"
 
 	// UncommittedStatefulOrderPlacementKeyPrefix is the key to retrieve an uncommitted stateful order and information
 	// about when it was placed. uncommitted orders are orders that this validator is aware of that have yet to be
 	// committed to a block and are stored in a transient store.
-	UncommittedStatefulOrderPlacementKeyPrefix = StatefulOrderKeyPrefix + "uncommitted/long_term/"
+	UncommittedStatefulOrderPlacementKeyPrefix = StatefulOrderKeyPrefix + "U/LT/"
 
 	// UncommittedStatefulOrderCancellationKeyPrefix is the key to retrieve an uncommitted stateful order cancellation.
 	// uncommitted cancelleations are cancellations that this validator is aware of that have yet to be
 	// committed to a block and are stored in a transient store.
-	UncommittedStatefulOrderCancellationKeyPrefix = "stateful_order_cancellation/uncommitted/long_term/"
+	UncommittedStatefulOrderCancellationKeyPrefix = "SOC/U/LT/"
 
 	// UncommittedStatefulOrderCountPrefix is the key to retrieve an uncommitted stateful order count.
 	// uncommitted orders are orders that this validator is aware of that have yet to be committed to a block and
 	// are stored in a transient store. This count represents the number of uncommitted stateful
 	// `placements - cancellations`.
-	UncommittedStatefulOrderCountPrefix = "stateful_order_count/uncommitted/long_term/"
+	UncommittedStatefulOrderCountPrefix = "Ct/U/LT/"
 
 	// StatefulOrderCountPrefix is the key to retrieve the stateful order count. The stateful order count
 	// represents the number of long term order placements and triggered conditional orders stored in state.
-	StatefulOrderCountPrefix = "stateful_order_count/long_term/"
+	StatefulOrderCountPrefix = "Ct/LT/"
```

Comment:
Nit: prefer not using `/` inside most keys

Nit: maybe consider `Num` or `Delta` instead of `Ct`